app/components-react/sidebar/PlatformIndicator.m.less

```diff
@@ -0,0 +1,71 @@
+// TODO: these are duplicated from NavTools.m.less, consider refactoring out if not needed there
+.platform-logo {
+  margin-right: 10px;
+
+  &-twitch {
+    color: var(--twitch) !important;
+  }
+  &-youtube {
+    color: var(--youtube) !important;
+  }
+  &-facebook {
+    color: var(--facebook) !important;
+  }
+  &-trovo, &-twitter {
+    width: 15px;
+    height: 15px;
+  }
+  &-tiktok {
+    color: var(--tiktok) !important;
+  }
+  &-streamlabs {
+    color: var(--teal) !important;
+  }
+  &-default {
+    color: var(--logged-in) !important;
+  }
+}
+
+.username {
+  margin-bottom: 0px;
+  flex-grow: 1;
+  line-height: 20px;
+  margin: 2px 0 0 0;
+}
+
+.login-arrow {
+  justify-self: flex-end;
+  margin-left: 5px;
+  transform: scaleX(-1);
+  -moz-transform: scaleX(-1);
+  -webkit-transform: scaleX(-1);
+  -ms-transform: scaleX(-1);
+  transition: all 0.2s ease-in-out;
+}
+
+.platform-icons {
+  display: flex;
+  flex-direction: row;
+  gap: 1px;
+  align-items: center;
+  justify-content: center;
+
+  // Override the zooming effect applied to sidebar icons
+  i {
+    &,
+    &:hover {
+      transform: none !important;
+      transition: none !important;
+    }
+  }
+  // Custom destination icon
+  // TODO: why does this need to be global?
+  :global(.fa-globe) {
```

Comment:
I understand that, I'm just not sure why it wouldn't apply to standard DOM elements like `.fa-globe` inside a parent that has the `platform-icons` class applied to it.

Reply:
i don't think i'm 100% clear about what you're asking, but here are how the two would compile to CSS if this helps to illustrate:
```
// LESS
.platform-icons {
  .fa-globe

// CSS
.platform-icons_sgshe534 .fa-globe_397sfe {
```
vs
```
// LESS
.platform-icons {
  :global(.fa-globe) {
  
// CSS
.platform-icons_sgshe534 .fa-globe {
```
so because fontawesome doesnt have a class named `fa-globe_397sfe` none of those styles would be applied

Reply:
Perfect, that makes sense, though I'd swear I've seen bare classes before (or perhaps it was only elements) working before. TIL.

Reply:
bare elements definitely work, and in any file appended `.g.less` bare classes will work, but all `.m.less` files are scoped

in legacy Vue files as well scoped bare classes will still work because the way Vue applies style scopes is insane KEKW